roles/wordpress-setup/tasks/nginx.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- include: nginx-client-cert.yml
+  tags: wordpress-setup-nginx-client-cert
+  notify: reload nginx
+
```

Comment:
Two minor requests:
* Could you move this `include` task right after the other [similar include tasks](https://github.com/roots/trellis/blob/f2b81074c83475837e544a8aa5c3e909e760aa8a/roles/wordpress-setup/tasks/main.yml#L2-L5) in `tasks/main.yml`? That placement strikes me as more consistent. I like the idea of avoiding [nested includes](https://www.google.com/search?q=ansible+"nested+includes") when it's easy.
* Let's omit the `notify` here. Technically, Nginx only needs to reload if the conf file produced by our `wordpress-site.conf.j2` changes. A change in the "Download client cert" task will trigger a `changed` [template task](https://github.com/roots/trellis/blob/f2b81074c83475837e544a8aa5c3e909e760aa8a/roles/wordpress-setup/tasks/nginx.yml#L33) which already has `notify: reload nginx`.